### PR TITLE
Feature/mobile view

### PR DESCRIPTION
**Pull request by ebonynon**
Fix mobile issues
~~past-event section is still pending~~
past-event section over to @Safnaj 
image
image
image

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -15,17 +15,17 @@ import {
   FontAwesomeModule,
   FaIconLibrary,
 } from '@fortawesome/angular-fontawesome';
-import { library } from '@fortawesome/fontawesome-svg-core';
-import { faFilm } from '@fortawesome/free-solid-svg-icons';
+// import { library } from '@fortawesome/fontawesome-svg-core';
+// import { faFilm } from '@fortawesome/free-solid-svg-icons';
```
Comment (Shehanka) [inline src/app/app.module.ts:19]: Remove these lines

File: src/app/app.module.ts [hunk 2]
```diff
@@ -15,17 +15,17 @@ import {
...
 import {
   faFacebook,
   faInstagram,
   faLinkedin,
   faTwitter,
   faYoutube,
-  faLinkedinIn,
+  //  faLinkedinIn,
```
Comment (Shehanka) [inline src/app/app.module.ts:26]: Remove these lines

File: src/app/components/common/footer/footer.component.html [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+        </div>
       </div>
+      <!--      <div class="col-md-4" style="padding-top: 0">-->
+      <!--        <img-->
+      <!--          class="navbar-brand"-->
+      <!--          src="https://i.postimg.cc/FRP7Q8kz/FOSS.png"-->
+      <!--          height="155"-->
+      <!--          alt="SLIIT FOSS Logo"-->
+      <!--        />-->
+      <!--      </div>-->
```
Comment (Shehanka) [inline src/app/components/common/footer/footer.component.html:26]: Remove these commented lines

File: src/app/components/common/navbar/navbar.component.html [hunk 4]
```diff
@@ -1,20 +1,31 @@
...
+  <!--    <img-->
+  <!--      class="navbar-brand"-->
+  <!--      src="https://i.postimg.cc/FRP7Q8kz/FOSS.png"-->
+  <!--      height="230"-->
+  <!--      width="220"-->
+  <!--    />-->
+  <!--  </div>-->
+  <!--  <div-->
+  <!--    class='d-flex align-content-start justify-content-start flex-wrap'-->
+  <!--  >-->
```
Comment (Shehanka) [inline src/app/components/common/navbar/navbar.component.html:27]: Remove these commented lines

File: src/app/components/member/member.component.html [hunk 5]
```diff
@@ -44,17 +47,17 @@ <h2 class="h2">What we do ?</h2>
...
+        <!--          <li><i class="fa fa-check"></i> Dolorem ratione dolorum</li>-->
+        <!--          <li>-->
+        <!--            <i class="fa fa-check"></i> Quo nihil natus ea non pariatur optio-->
+        <!--            occaecati-->
+        <!--          </li>-->
+        <!--          <li>-->
+        <!--            <i class="fa fa-check"></i> Duis aute irure dolor in reprehenderit-->
+        <!--            in-->
+        <!--          </li>-->
+        <!--        </ul>-->
```
Comment (Shehanka) [inline src/app/components/member/member.component.html:60]: Remove these commented lines

File: src/app/components/past-events/past-events.component.html [hunk 6]
```diff
@@ -3,26 +3,30 @@
...
-            >
-              <img src="{{ slide.img }}" class="card-img-top" />
-              <div class="card-body">
-                <h5 class="card-title">{{ slide.title }}</h5>
-                <p class="card-text">
-                  {{ slide.description }}
-                </p>
-              </div>
-              <button class="member-btn">View More</button>
+    <!--    <owl-carousel-o [options]="customOptions">-->
```
Comment (Shehanka) [inline src/app/components/past-events/past-events.component.html:6]: Why did you comment this line?

File: src/index.html [hunk 7]
```diff
@@ -9,24 +9,31 @@
...
+<link
+  rel="stylesheet"
+  href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+  integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
+  crossorigin="anonymous"
+/>
+<link
+  rel="stylesheet"
+  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
+  integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA=="
```
Comment (Shehanka) [inline src/index.html:22]: Remove these commented lines

**Comment by ebonynon** [comment]
Also need the clean the code

File: src/app/components/common/footer/footer.component.html [hunk 8]
```diff
@@ -1,15 +1,29 @@
 <!-- Footer -->
-<footer class="page-footer font-small bg-dark pt-4">
+<footer class="page-footer font-small pt-4">
```
Comment (Shehanka) [inline src/app/components/common/footer/footer.component.html:2]: Why did u remove *bg-dark* ?

**Review by Shehanka — CHANGES_REQUESTED** [review]

File: src/app/components/common/footer/footer.component.html (hunk 8 above)
Comment (ebonynon) [reply src/app/components/common/footer/footer.component.html:2]: To make footer color black

**Comment by Safnaj** [comment]
@ebonynon make sure u always get the latest changes before u commit. Pls share a screenshot or video

File: src/app/components/past-events/past-events.component.html (hunk 6 above)
Comment (ebonynon) [reply src/app/components/past-events/past-events.component.html:6]: I commented it

**Comment by ebonynon** [comment]
> @ebonynon make sure u always get the latest changes before u commit. Pls share a screenshot or video

I'm done. Now you can merge
cc: @Shehanka

**Comment by Safnaj** [comment]
@Shehanka can u quickly review this and merge. I have one PR pending

**Comment by Safnaj** [comment]
@ebonynon why the images missing in the mobile view ?

**Comment by ebonynon** [comment]
> @ebonynon why the images missing in the mobile view ?

https://github.com/sliit-foss/sliitfoss/blob/508ab9507d92868a37549b12fd9a2fd9f8e724e0/src/app/components/member/member.component.html#L16

https://github.com/sliit-foss/sliitfoss/blob/508ab9507d92868a37549b12fd9a2fd9f8e724e0/src/app/components/member/member.component.html#L27

I set images auto hide for xs n sm screen sizes as there in no way to fix images overlapping issue in mobile view .

**Comment by Safnaj** [comment]
@ebonynon images should be there for mobile views too. Use `col-md-7 `instead of `col-7`

**Review by Shehanka — APPROVED** [review]

**Pull request merged**